examples/sentence-transformers-training/sts/readme.md

```diff
@@ -0,0 +1,95 @@
+# Semantic Textual Similarity
+
+Semantic Textual Similarity (STS) assigns a score on the similarity of two texts. In this example, we use the [stsb](https://huggingface.co/datasets/sentence-transformers/stsb) dataset as training data to fine-tune our model. See the following example scripts how to tune SentenceTransformer on STS data:
+
+- **[training_stsbenchmark.py](training_stsbenchmark.py)** - This example shows how to create a SentenceTransformer model from scratch by using a pre-trained transformer model (e.g. [`distilbert-base-uncased`](https://huggingface.co/distilbert/distilbert-base-uncased)) together with a pooling layer.
+
+
+## Requirements
+
+First, you should install the requirements:
+```bash
+pip install -U sentence-transformers
+pip install git+https://github.com/huggingface/optimum-habana.git
+```
+
+
+## Usage
+
+To fine training on STS you can do following steps -
```

Comment:
To fine-tune, also you may not expand STS as the whole sentence

Reply:
There 3 examples are copied from sentence-transformer training example including readme file besides the command using by hpu. So it's better to keep consistent with original ST example version. if you don't agree i can change into like language-model version. Thanks.

Reply:
Original README https://github.com/UKPLab/sentence-transformers/blob/master/examples/training/sts/README.md  is consistent with language_modeling README. You can follow either of them.